__tests__/app.test.js

```diff
@@ -262,3 +270,68 @@ describe("POST /api/articles/:article_id/comments", () => {
       })
   })
 })
+describe("PATCH /api/articles/:article_id", () => {
+  test("201: Responds with updated article vote count", () => {
+    return request(app)
+      .patch("/api/articles/2")
+      .send({ inc_votes: 1 })
+      .expect(201)
+      .then(({ body: { article } }) => {
+        expect(article.votes).toBe(1)
+      })
+  })
+  test("201: Returns with updated article vote count", () => {
+    return request(app)
+      .patch("/api/articles/1")
+      .send({ inc_votes: 5 })
+      .expect(201)
+      .then(({ body: { article } }) => {
+        expect(article).toMatchObject({
+          article_id: 1,
+          author: expect.any(String),
+          title: expect.any(String),
+          body: expect.any(String),
+          topic: expect.any(String),
+          created_at: expect.any(String),
+          votes: 5,
+          article_img_url: expect.any(String),
+        })
+      })
+  })
+  test("400: Responds with an error if sent a empty object", () => {
+    return request(app)
+      .patch("/api/articles/2")
+      .send({})
+      .expect(400)
+      .then(({ body: { message } }) => {
+        expect(message).toBe("Bad request")
+      })
+  })
+  test("400: Responds with an error if request key is not inc_votes", () => {
+    return request(app)
+      .patch("/api/articles/1")
+      .send({ votes: 1 })
+      .expect(400)
+      .then(({ body: { message } }) => {
+        expect(message).toBe("Bad request")
+      })
+  })
+  test("400: Responds with an error message if inc_vote is not a number", () => {
+    return request(app)
+      .patch("/api/articles/5")
+      .send({ inc_votes: "banana" })
+      .expect(400)
+      .then(({ body: { message } }) => {
+        expect(message).toBe("Bad request")
+      })
+  })
+  test("400: Article_id does not exist", () => {
+    return request(app)
+      .patch("/api/articles/banana")
+      .send({ inc_votes: 10 })
+      .expect(400)
+      .then(({ body: { message } }) => {
+        expect(message).toBe("Bad request")
+      })
```

Comment:
Really brilliant testing so far! We're just missing one more test though- what would happen if someone tried to patch` /api/articles/9999` ?